lib/modules/datasource/npm/get.ts

```diff
@@ -67,6 +68,134 @@ const PackageSource = z
   ])
   .catch({ sourceUrl: null, sourceDirectory: null });
 
+const DepResponse = z
+  .object({
+    versions: LooseRecord(
+      z.object({
+        repository: PackageSource,
+        homepage: z.string().nullish().catch(null),
+        deprecated: z.string().nullish().catch('Unknown deprecation reason'),
```

Comment:
catch will only be used if it's a wrong type that the expected string?

Reply:
I'm not sure if `deprecated` can be just `true`, so if it is, it will be converted to this default string

Reply:
BTW if it's `false` then it will be converted to this string too, but I don't worry about `false` as much as I worry about `true`.

Reply:
maybe better allow bool and transform to string if true and log debug if false?